contrib/vagrant/provision-util.sh

```diff
@@ -128,6 +128,9 @@ os::provision::set-os-env() {
   fi
   os::provision::set-bash-env "${origin_root}" "${config_root}" \
 "/root/${file_target}"
+
+  # Make symlinks to the bash completions for the openshift commands
+  ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
```

Comment:
Don't we also need to source the files under /etc/bash_completion.d to enable the tab completion?